pdf-converter/README.md

```diff
@@ -0,0 +1,105 @@
+This is NOT an official Google product.
+
+# Prepare for installing
```

Comment:
Can you make this into a `Dockerfile` so we can create the CE instance using that container image?

Reply:
@davidcavazos  Created Dockerfile and deployed on CE as the container. Updated README.md file as well.

Reply:
Thank you! Merging